### PR TITLE
Added Fork of PolyMC credit

### PR DESCRIPTION
**Pull request by 32bitx64bit**
This is technically a fork of PolyMC and should be credited (Even tho what the developer did wasn't right.)

Signed-off-by: 32bitx64bit <86190528+32bitx64bit@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 Prism Launcher is a custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once.
 
-This is a **fork** of the MultiMC Launcher and not endorsed by MultiMC.
+This is a **fork** of the PolyMC Launcher which is a fork of Multimc and is not endorsed by PolyMC or Multimc.
```
Comment (DioEgizio) [inline README.md:8]:
```suggestion
This is a **fork** of the PolyMC Launcher which is a fork of MultiMC and it's not endorsed by either PolyMC nor MultiMC.
```

**Comment by 32bitx64bit** [comment]
Clutch fix m8, thanks.

**Comment by Scrumplex** [comment]
I don't want to mention PolyMC on the README. The only reason the message is there, is because MultiMC specifically asks for it.

FOSS isn't about GitHub organizations or repositories. It is about merit. PolyMC effectively rebranded on that day, because the thing that's called PolyMC right now isn't the same thing as it was before October 17.

We are keeping PolyMC in our copyright headers for legal reasons already.

**Comment by 32bitx64bit** [comment]
I understand you not wanting to add but maintainer of PolyMC might want it in the feature

**Comment by BeigeAnimal** [comment]
> I understand you not wanting to add but maintainer of PolyMC might want it in the feature

doubtful, even if they did given their recent actions they don't really deserve it and so just let prism distance itself from that as much as possible.

**Comment by Aeris1One** [comment]
> I understand you not wanting to add but maintainer of PolyMC might want it in the feature

Also, unlike MultiMC, PolyMC is under GPL, so Prism needs to follow GPL, nothing more. PolyMC maintainer has no right on Prism except thoses in GPL (like crediting in license and headers)

**Pull request closed without merging**